File: app/src/main/java/com/drkein/stayon/act/EmptyActivity.java
package com.drkein.stayon.act;

import android.app.Activity;
import android.os.Bundle;
import android.os.Handler;

/**
 * @author kein
 *
 */
public class EmptyActivity extends Activity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
               finish();
            }
        }, 500);
    }


}

File: app/src/main/java/com/drkein/stayon/act/EmptyActivity.kt
package com.drkein.stayon.act

import android.app.Activity
import android.os.Bundle
import android.os.Handler

/**
 * @author kein
 */
class EmptyActivity : Activity() {

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Handler().postDelayed({ finish() }, 500)
    }

}